scripts/deploy.sh

```
@@ -6,77 +6,104 @@ cd "$APP_DIR"
+    source "$OTEL_ENV_FILE"
+    set +a
+    JAVA_AGENT_OPTS="-javaagent:$OTEL_AGENT_JAR"
+  else
+    echo "> [$PORT] Grafana OTel agent 미적용"
+  fi
+
+  START_LINE=$(wc -l < "$LOG_FILE" 2>/dev/null || echo 0)
+
+  nohup java $JAVA_AGENT_OPTS -jar \
+    -Duser.timezone=Asia/Seoul \
+    $JAR_PATH \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)